autoscheduler/frontend/src/components/SchedulingPage/CourseSelectColumn/CourseSelectCard/ExpandedCourseCard/SectionSelect/SectionSelect.tsx

```diff
@@ -100,34 +100,30 @@ const SectionSelect: React.FC<SectionSelectProps> = ({ id }): JSX.Element => {
     const remainingSeatsColor = remainingSeats > 0 ? 'black' : 'red';
     // show section number and remaining seats if this is the first meeting for a section
     return (
-      <>
-        <Typography className={styles.denseListItem}>
-          <span>
-            {section.sectionNum}
-          </span>
-          <span className={styles.rightAlignedText} style={{ color: remainingSeatsColor }}>
-            {`${remainingSeats}/${section.maxEnrollment} seats left`}
-          </span>
-        </Typography>
-      </>
+      <Typography className={styles.denseListItem} component="tr">
+        <td>
+          {section.sectionNum}
+        </td>
+        <td
+          style={{ color: remainingSeatsColor, textAlign: 'right' }}
+          colSpan={3}
+        >
+          {`${remainingSeats}/${section.maxEnrollment} seats left`}
+        </td>
+      </Typography>
     );
   };
 
   const renderMeeting = (mtg: Meeting, showSectionNum: boolean): JSX.Element => (
-    <React.Fragment key={mtg.id}>
+    <>
```

Comment:
![image](https://user-images.githubusercontent.com/28655462/90984703-4cf2cb00-e53c-11ea-8a89-8756731c720c.png)
I believe that you removing the `key` here is causing this error, which shows up whenever the section tables are created